.github/workflows/spec.yml

```diff
@@ -0,0 +1,19 @@
+name: "Rspec"
+on:
+  pull_request:
+    branches: [ "main" ]
+jobs:
+  rspec:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Install Ruby and gems
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.1.2
+          bundler-cache: true
+      - name: Run tests
+        run: bundle exec rspec spec
```

Comment:
**nit:** `bundle exec rspec` is sufficient

Reply:
Are you sure?
I don't think is sufficient because the gem are installed under `vendor/` and in this way it's executing also the specs for them.

ref https://github.com/nebulab/deprecations_detector/actions/runs/5719646130/job/15497987360

Reply:
Hmm possible. Better to explicitly mention the `spec` dir then